src/main/java/by/exadel/internship/controller/InternshipController.java

```diff
@@ -31,8 +33,19 @@
 
     @GetMapping("/{internshipId}")
     @ApiOperation("return internship by id")
-    public GuestFullInternshipDTO getInternshipById(@PathVariable("internshipId") UUID internshipId) {
-        return internshipService.getById(internshipId);
+    public ResponseEntity<?> getInternshipById(@PathVariable("internshipId") UUID internshipId, boolean isAdminInformation) {
```

Comment:
```suggestion
    public ResponseEntity<?> getInternshipById(@PathVariable("internshipId") UUID internshipId, boolean fullRepresentation) {
```

Reply:
it is beter,  may i use "is" at the beginning of the word?

Reply:
done